R/utils-xportr.R

```diff
@@ -138,6 +161,12 @@ fmt_labs <- function(x) {
   glue("{labs} {encode_vals(val)}")
 }
 
+fmt_fmts <- function(x) {
+  fmts <- ntext(length(x), "Format", "Formats")
+  glue("{fmts} {encode_vals(x)}")
+}
+
+
```

Comment:
is this used somewhere?  I see it is new but don't see it being used?

Reply:
Used up around line 114

Reply:
oh lol i see can you move it so it comes before that call

Reply:
If you scroll up a bit, you'll see a few other functions that are also mentioned after where they are called, do you want me to reorganize the entirety of the file such that functions are created **before** where they are called?

Reply:
Please just to increase readability for those who come afterwards.